backend/db/seed.sql

```diff
@@ -456,136 +456,224 @@ INSERT INTO events
+
+
+    -- FUTURE ==============
+    (   CURRENT_DATE + INTERVAL '2 days' + TIME '13:00-04', --'2020-07-01 13:00-04',
+        CURRENT_DATE + INTERVAL '2 days' + TIME '13:45-04', --'2020-07-01 13:45-04',
+        'Let''s Do Remote Lunch!',
+        'Spend your WFH lunch break with 6-10 of our Fellows! Chat about anything! Get to know our Fellows and share your path to tech. We''ll share a list of possible topics as well. We''ll have 3 "Lunch Rooms" over Zoom each day so please choose more than 1 day in case dates get filled up. I''ll reach back out to confirm your date.',
+        1, 'Pursuit HQ: 47-10 Austell Place, 2nd Fl Long Island City, NY 11101',
+        'Jessica Shyu', 6, FALSE  ),
+
+    (   CURRENT_DATE + INTERVAL '4 days' + TIME '10:00-04',
+        CURRENT_DATE + INTERVAL '4 days' + TIME '10:45-04',
+        'Let''s Do Remote Breakfast!',
+        'Spend a WFH breakfast break with 6-10 of our Fellows! Chat about anything! Get to know our Fellows and share your path to tech. We''ll share a list of possible topics as well. We''ll have 3 "Breakfast Rooms" over Zoom each day so please choose more than 1 day in case dates get filled up. I''ll reach back out to confirm your date.',
+        1, 'Pursuit HQ: 47-10 Austell Place, 2nd Fl Long Island City, NY 11101',
+        'Jessica Shyu', 6, FALSE  ),
+
+    (   CURRENT_DATE + INTERVAL '11 days' + TIME '10:00-04', --'2020-07-13 10:00-04',
+        CURRENT_DATE + INTERVAL '11 days' + TIME '15:00-04', --'2020-07-13 15:00-04',
+        'Pursuit + ACME DTC Host the 6.2 FullStack Web Technical Showcase',
+        'After several intensive months of training, Pursuit’s 6.2 FullStack Web fellows have spent the past 8 weeks working with volunteer mentors to build their capstone projects - original and innovative web applications from ideation to deployment. The Technical Showcase is an opportunity for Fellows to present their projects to industry professionals and receive technical feedback on their projects and pitches.  We are thrilled to partner with Clorox DTC on this event to celebrate the accomplishments of the 6.2 FullStack Web Fellows and further our shared goal of creating access to opportunity for everyone.  Join us to celebrate!',
+        10, 'Zoom: https://zoom.us/my/alejos',
+        'Dessa Shepherd', 10, TRUE  );
 
         
-
-
-
 INSERT INTO event_volunteers
     (eventv_id, volunteer_id, confirmed, volunteered_time)
 VALUES
-    (1, 1, TRUE, 2),
-    (1, 2, TRUE, 3),
-    (1, 10, TRUE, 1),
-    (1, 6, TRUE, 1),
-    (2, 2, TRUE, 2),-- 5
+    (1, 1, TRUE, 8),
+    (1, 2, TRUE, 8),
+    (1, 10, TRUE, 8),
+    (1, 6, TRUE, 8),
+    (2, 2, TRUE, 2),
     (2, 3, TRUE, 3),
     (3, 1, TRUE, 1),
-    (4, 2, FALSE, 0),
-    (4, 3, FALSE, 0),
-    (5, 4, FALSE, 0),-- 10
-    (5, 5, FALSE, 0),
-    (5, 6, TRUE, 1),
-    (5, 7, FALSE, 0),
-    (6, 8, FALSE, 0),
-    (6, 9, FALSE, 0),-- 15
-    (7, 10, FALSE, 0),
-    (7, 6, TRUE, 0),
+    (4, 2, FALSE, 3),
```

Comment:
I believe better set all `volunteered_time` to `0` where `confirmed` is `false`
Technically, someone who's not confirmed to an event can't have hours